docs/build-the-docs.md

```diff
@@ -1,15 +1,96 @@
 # Building the Documentation
+[link-git]: https://git-scm.com
+[link-Node-8]: https://nodejs.org
+[link-Gulp-CLI]: http://gulpjs.com
+[link-Yarn]: https://yarnpkg.com
+[link-git-package]: https://git-scm.com/downloads
+[link-nvm]: https://github.com/creationix/nvm
+[link-nvm-installation-instructions]: https://github.com/creationix/nvm#installation
+[link-open-windows-cmd-prompt]: https://www.lifewire.com/how-to-open-command-prompt-2618089
 
-## Install Antora's Dependencies
+## Install the Prerequisites
 
-Before you can build the documentation, you need to install the required dependencies. To install them, you need to run `yarn install`. This will install all the dependencies specified in `package.json`.
+Before you can build the ownCloud documentation, you need to install the following software:
 
-**Note:** If your environment already has GNU make installed, you can run `make setup` instead.
+- [git][link-git] (command: `git`)
+- [Node 8][link-Node-8] (command: `node`)
+- [Gulp CLI][link-Gulp-CLI] (command: `gulp`)
```

Comment:
Gulp is not needed for the documentation, it's only needed to build a custom theme.

Reply:
Besides that, gulp gets already installed via yarn within the ui repo.